.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,83 @@
+```
+┌─────────────┬──────────┬──────────┬───────┬─────────────────────────┐
+│ Workflow    │ Before   │ After    │ Trend │ Performance             │
+├─────────────┼──────────┼──────────┼───────┼─────────────────────────┤""".format(
+        before_sparkline=generate_sparkline(data['baseline']),
+        before_avg=sum(data['baseline'])/len(data['baseline']),
+        after_sparkline=generate_sparkline(data['current']),
+        after_avg=sum(data['current'])/len(data['current'])
+    )
+
+    for name, metrics in data['workflows'].items():
+        change = ((metrics['before'] - metrics['after']) / metrics['before']) * 100
```

Flake8 (7.2.0)

[error] local variable 'change' is assigned to but never used

(F841)